docs/configuration.asciidoc

```diff
@@ -185,6 +185,41 @@ please https://github.com/elastic/apm-agent-nodejs/issues[open an issue].
 
 Note that not all core Node.js API's can be instrumented without the use of Async Hooks if running Node.js 8 or above.
 
+[[transaction-ignore-urls]]
+==== `transactionIgnoreUrls`
+
```

Comment:
Good call out.  If I'm reading that right "Dynamic" configuration means users can confg this via central config. 

How about we hold off this for now? My thinking is this configuration isn't available yet in the 7.10 release, and agents release off cycle from mainline elastic.  Once the `TRANSACTION_IGNORE_URL` setting makes it into Kibana/APM-UI proper we can come back and mark this as dynamic. 

Thoughts? (if so, just unresolved and we can keep talking)

Reply:
@bmorelli25 Thoughts on this? Is there a way we could say "in APM server version 7.11" alongside the "dynamic configuration" badge/link?

Reply:
Good catch. Whether we wait on adding this badge to the docs or not, it's probably important either way to mention that stack version 7.11+ is required. I think text next to the badge is fine. We could add it to the badge too, but that might be overkill. . . 

<img width="226" alt="Screen Shot 2020-11-18 at 3 52 58 PM" src="https://user-images.githubusercontent.com/5618806/99602399-3ed70e00-29b6-11eb-99ab-2afb27bae3b9.png">

Reply:
@bmorelli25 Ooh, I like that. What's the asciidoc syntax for that?

Reply:
Badges are a bit of a hack in our build as we don't have native support for them. In this case, the new [badge](https://img.shields.io/badge/Dynamic-Requires%20APM%20Server%207.11%2B-blue) needs to be added to the images directory and then referenced:

`<<dynamic-configuration, image:./images/dynamic-config-stack-711.svg[] >>`

I will say-- after thinking some more on this, I don't think it makes to add this badge to the docs yet; I think it will only confuse users.

Reply:
Thanks @bmorelli25 -- we've captured our intent to update this label once the feature lands in kibana: https://github.com/elastic/apm-agent-nodejs/issues/1883